docker/Dockerfile

```diff
@@ -0,0 +1,43 @@
+FROM debian:buster-slim
+
+ENV DEBIAN_FRONTEND noninteractive
+
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends \
+    python2-minimal=2.7.16-1 \
+    python-pip=18.1-5 \
+    python-setuptools=40.8.0-1 \
+    ca-certificates=20190110 \
+    nodejs=10.15.2~dfsg-2 \
+    npm=5.8.0+ds6-4 \
+    git=1:2.20.1-2 \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/* \
```

Comment:
Codacy is slacking in regard of updating linter config files (or we should have it here instead)
Per https://docs.docker.com/develop/develop-images/dockerfile_best-practices/
> Official Debian and Ubuntu images [automatically run](https://github.com/moby/moby/blob/03e2923e42446dbb830c654d0eec323a0b4ef02a/contrib/mkimage/debootstrap#L82-L105) `apt-get clean`, so explicit invocation is not required.